android/build.gradle

```diff
@@ -1,10 +1,63 @@
 import com.android.Version
+import groovy.json.JsonSlurper
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+def getReactNativeVersion() {
+    def packageJsonFile = file("$rootDir/../package.json")
```

Comment:
Package.json is not necessarily in ../ is there any other way to retrieve the version ? I am also wondering if some frameworks can hide react-native in sub package 🤨